acceptance/run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd $(dirname $0)/..
+build/builder.sh make install
```

Comment:
Yeah, I noticed this wasn't working right before I left work today. It should work. It was working at one time. I'll see what broke. It is definitely possible to have a container write to a directory that was mapped in from outside boot2docker. This is how `build/builder.sh` works.